lib/_stream_writable.js

```diff
@@ -728,5 +735,12 @@ Object.defineProperty(Writable.prototype, 'writableFinished', {
 Writable.prototype.destroy = destroyImpl.destroy;
 Writable.prototype._undestroy = destroyImpl.undestroy;
 Writable.prototype._destroy = function(err, cb) {
+  const state = this._writableState;
+  for (let entry = state.bufferedRequest; entry; entry = entry.next) {
+    entry.callback(new ERR_STREAM_DESTROYED('write'));
```

Comment:
This might need to be semver major due to this change?

Reply:
@mcollina: Does this need to be invoked in next tick? Or is `_destroy` already in next tick?

Reply:
ping @mcollina

Reply:
I don’t think we should call this in a nextTick